openfecli/commands/plan_rhfe_network.py

```diff
@@ -70,8 +73,10 @@ def plan_rhfe_network_main(
     help=OUTPUT_DIR.kwargs["help"] + " Defaults to `./alchemicalNetwork`.",
     default="alchemicalNetwork",
 )
+@N_PROTOCOL_REPEATS.parameter(multiple=False, required=False, default=3, help=N_PROTOCOL_REPEATS.kwargs["help"])
```

Comment:
I don't love that we have default=3 assigned at both the CLI level and the protocol level, but I think this is the clearest way to handle it on the CLI.